src/Core/System/SalesChannel/Context/SalesChannelContextPersister.php

```diff
@@ -4,18 +4,19 @@
 
 use Doctrine\DBAL\Connection;
 use Shopware\Core\Framework\Util\Random;
-use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Shopware\Core\System\SalesChannel\Event\SalesChannelContextTokenChangeEvent;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
 
 class SalesChannelContextPersister
 {
-    /**
```

Comment:
please do not remove the property annotations. also add annotations to your new properties